tools/summonerd/src/web.rs

```diff
@@ -40,7 +40,7 @@ async fn serve_css() -> impl IntoResponse {
         .status(StatusCode::OK)
         .header("Content-Type", "text/css")
         .header("Cache-Control", "public, max-age=3600") // Cache for 1 hour
-        .body(axum::body::Full::from(css))
+        .body(axum::body::Body::from(css))
```

Comment:
`axum::body::Full` was removed, and one is to use `Body` instead (according to their changelog)